src/presets/vercel.ts

```diff
@@ -163,50 +181,60 @@ function generateBuildConfig(nitro: Nitro) {
           continue: true,
         })),
       { handle: "filesystem" },
-      // ISR rules
-      ...rules
-        .filter(
-          ([key, value]) =>
-            // value.isr === false || (value.isr && key.includes("/**"))
-            value.isr !== undefined
-        )
-        .map(([key, value]) => {
-          const src = key.replace(/^(.*)\/\*\*/, "(?<url>$1/.*)");
-          if (value.isr === false) {
-            // we need to write a rule to avoid route being shadowed by another cache rule elsewhere
-            return {
-              src,
-              dest: "/__nitro",
-            };
-          }
+    ],
+  });
+
+  // Early return without serverless functions if we are using a static preset
+  if (nitro.options.preset === "vercel-static") {
```

Comment:
(with that change, this could be like `if (nitro.options.static)` in a more generic way

Reply:
ah, yes, this should have been instead `!nitro.options.build` like in netlify preset.